src/com/moclojer/specs/moclojer.clj

```diff
@@ -47,13 +51,15 @@
         :body (render-template (:body webhook-config) request)
         :headers (:headers webhook-config)
         :sleep-time (:sleep-time webhook-config)}))
-    {:body    (build-body response request)
-     :status  (:status response)
-     :headers (into
-               {}
-               (map (fn [[k v]]
-                      [(name k) (str v)]))
-               (:headers response))}))
+    (let [body (build-body response (:parameters request))]
+      (log/log :info :body (json/read-str body :key-fn keyword))
+      {:body  body
+       :status 200
+       :headers (into
+                 {}
+                 (map (fn [[k v]]
+                        [(name k) (str v)]))
+                 (:headers response))})))
 
```

Comment:
does destructing [k v] account for repeating headers?

would `map` returning ["something" 1 "something" 2] be treated without errors?